tests/unit/test_blockchain_economics_model.py

```diff
@@ -47,22 +47,26 @@ def test_rough_economics():
     assert float(round(e.erc20_total_supply / Decimal(1e9), 2)) == 3.89  # As per economics paper
 
     # Check that we have correct numbers in day 1 of the second phase
-    initial_rate = (e.erc20_total_supply - int(e.first_phase_total_supply)) * (e.lock_duration_coefficient_1 + 365) / \
-                   (e.issuance_decay_coefficient * e.lock_duration_coefficient_2)
+    initial_rate = (e.erc20_total_supply - int(e.first_phase_total_supply)) \
+        * (e.lock_duration_coefficient_1 + 365) \
+        / (e.issuance_decay_coefficient * e.lock_duration_coefficient_2)
     assert int(initial_rate) == int(e.first_phase_max_issuance)
 
-    initial_rate_small = (e.erc20_total_supply - int(e.first_phase_total_supply)) * e.lock_duration_coefficient_1 / \
-                         (e.issuance_decay_coefficient * e.lock_duration_coefficient_2)
+    initial_rate_small = (e.erc20_total_supply - int(e.first_phase_total_supply))\
+        * e.lock_duration_coefficient_1 \
+        / (e.issuance_decay_coefficient * e.lock_duration_coefficient_2)
     assert int(initial_rate_small) == int(initial_rate / 2)
 
     # Sanity check that total and reward supply calculated correctly
     assert int(LOG2 / (e.token_halving * 365) * (e.erc20_total_supply - int(e.first_phase_total_supply))) == int(initial_rate)
     assert int(e.reward_supply) == int(e.erc20_total_supply - Decimal(int(1e9)))
 
-    # Sanity check for lock_duration_coefficient_1 (k1), issuance_decay_coefficient (d) and lock_duration_coefficient_2 (k2)
-    assert e.lock_duration_coefficient_1 * e.token_halving == \
-           e.issuance_decay_coefficient * e.lock_duration_coefficient_2 * LOG2 * e.small_stake_multiplier / 365
-
+    with localcontext() as ctx:  # TODO: Needs follow up - why the sudden failure (python 3.8.0)?
```

Comment:
Preemptively - I'd like to indicate this addition also... 

This is a badly behaving PR!  No Idea why this started to fail suddenly.  (very small error)